runtime/onert/frontend/base_loader/include/base_loader.h

```diff
@@ -1702,6 +1715,9 @@ void BaseLoader<LoaderDomain, SpecificLoader>::loadOperation(const Operator *op,
     case BuiltinOperator::BuiltinOperator_TILE:
       loadTile(op, subg);
       return;
+    case BuiltinOperator::BuiltinOperator_loadFusedBatchNorm:
```

Comment:
We will not use tflite and circle builtin operation for `FusedBatchNorm` operation.

Reply:
I will modify to call through loadCustom() referring to #1449.